pypit/arcomb.py

```diff
@@ -18,17 +18,18 @@ def comb_frames(frames_arr, det, method='weightmean', spect=None, weight=None, f
     method: How to combine? Allowed values are 'mean', 'median', 'weightmean'
     weight: How to weight the combination ('counts', 'exptime', None)
     frametype: What type of frame are you combining? (only used for screen printout)
-    rej_cosmicray: Sigma level to reject cosmic rays (< 0.0 means no CR removal)
-    rej_lowhigh: Number of low/high pixels to reject, [low,high]
-    rej_level: Rejection level (in standard deviations), where < 0.0 means no rejection [low,high]
-    sat_pix: What to do with saturated pixels (options are: 'reject', 'force', 'nothing')
-    set_allrej: What to do if all pixels are rejected (options are: 'min', 'max', 'mean', 'median', 'weightmean')
+    satpix: What to do with saturated pixels (options are: 'reject', 'force', 'nothing')
     maskvalue: What should the masked values be set to (should be greater than the detector's saturation value -- Default = 1 + 2**20)
+    reject: Dictionary of rejection options
     """
     from pypit import arcycomb
+    dnum = 'det{0:02d}'.format(det)
     ###########
     # FIRST DO SOME CHECKS ON THE INPUT
     ###########
+    # Check the reject dictionary
+    if reject is None:
```

Comment:
Many of these parameters are now not needed, because the settings are global. I've made the changes to this routine and throughout the code.